src/main/java/com/google/sps/data/FavoriteRecipe.java

```diff
@@ -21,15 +21,17 @@
   private long favoriteId;
   private String userId;
   private long recipeId;
+  private String dateFavorited;
```

Comment:
You may want to use DateProperty or  DateTimeProperty instead. This will give you the flexibility to make datastore queries with date time condition.